src/api/middleware/validators/offer.js

```diff
@@ -146,7 +168,7 @@ const jobMinDurationEditable = async (jobMinDurationCandidate, { req }) => {
         const { jobMaxDuration: jobMaxDurationCandidate } = req.body;
 
         // If the new publishMinDuration is after the new publishMaxDuration, the verification will be done in publishMaxDurationEditable
-        if (jobMinDurationCandidate >= offer.jobMaxDuration.toString() &&
+        if (jobMinDurationCandidate > offer.jobMaxDuration.toString() &&
```

Comment:
I just noticed that this would fail for offers that don't have a jobMaxDuration (possible since it is optional) since undefined.toString() throws an error

Make it 

```js
if ((offer.jobMaxDuration && jobMinDurationCandidate > offer.jobMaxDuration.toString()) && !jobMaxDurationCandidate)
```